### ISSUE TITLE
Add web kms integration

### ISSUE THREAD

**Issue by OR13** [transmute-industries/did-wallet#5]
https://github.com/w3c-ccg/webkms

**Comment by jandrieu** [comment]
@OR13 Any plans for a WebKMS implementation from Transmute?

**Comment by OR13** [comment]
@jandrieu yes, but ideally after we can see Digital Bazaar and Secure Key's implementations passing interop tests :)
